tutorials/fundamentals/regexp-starts-with.md

```diff
@@ -0,0 +1,14 @@
+To check if a string starts with a regular expression, use the `startsWith()` function.
+Regular expressions begin and end with a `/` but can have different characters in between, like `^`.
```

Comment:
Also not true. `'/ab+c/'` is a string, not a regular expression. `/ab+c/` is a regular expression.